docs/ref/migration-operations.txt

```diff
@@ -345,6 +345,12 @@ set ``atomic=False``.
     ``schema_editor.connection.alias``, where ``schema_editor`` is the second
     argument to your function).
 
+.. warning::
```

Comment:
I think warning is a bit strong. I try to avoid call out boxes since they are somewhat disruptive to the flow, but it seems appropriate here. ``note`` is probably enough though.